server/src/test/java/org/candlepin/dto/api/v1/ConsumerDTOTest.java

```diff
@@ -94,7 +99,9 @@ public ConsumerDTOTest() {
         this.values.put("EntitlementStatus", "test-entitlement-status");
         this.values.put("ServiceLevel", "test-service-level");
         this.values.put("Usage", "test-usage");
-        this.values.put("Offering", "test-offering");
+        this.values.put("Role", "test-role");
+        this.values.put("SystemPurposeStatus", "matched");
+        this.values.put("AddOns", addOns);
```

Comment:
Need to similarly update ConsumerTranslatorTest.java to set and verify the 4 new fields there too (unfortunately the test framework doesn't fail automatically if you don't set the new fields manually on the test, so its something we have to remember all the time).

Reply:
Done. Will be included in next update of this PR. Thanks!